quadkey/bq/sql/LONGLAT_ASQUADINTLIST_RESOLUTION.sql

```diff
@@ -14,16 +14,16 @@ AS """
     if (longitude === undefined || longitude === null || latitude === undefined || latitude === null) {
         return null;
     }
-    const zoom_min = parseInt(__zoom_min);
-    const zoom_max = parseInt(__zoom_max);
-    const zoom_step = parseInt(__zoom_step);
-    const resolution = parseInt(__resolution);
+    const zoomMin = parseInt(zoom_min);
+    const zoomMax = parseInt(zoom_max);
+    const zoomStep = parseInt(zoom_step);
+    const _resolution = parseInt(resolution);
```

Comment:
What? So you removed it from the SQL parameter and then added it here which forces you to add it in 3 other places?

Reply:
True, was mainly interested in removing "_" form the function notation rather than the inside and didn't came with a good name